test/unit/search/fm_index/bi_fm_index_dna4_test.cpp

```diff
@@ -5,11 +5,16 @@
 // shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 // -----------------------------------------------------------------------------------------------------
 
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
hmm. should we define this locally in tests or maybe globally?

Reply:
I'd do it locally since we usually want deprecation warnings - except when we test the deprecated stuff.
If I put the pragma one line of the test, it throws warnings.

Reply:
alright I guess we just need to keep track of those when we remove the deprecated stuff for the next release, that we remove the macro too